test/schema/me/conversation/index.js

```diff
@@ -121,6 +123,8 @@ describe("Me", () => {
           from: {
             email: "fancy_german_person@posteo.de",
           },
+          is_last_message_to: true,
+          last_message_open: null,
```

Comment:
And what does this look like when it's not `null`? Doesn't need a test necessarily if it's already addressed elsewhere, but I'm genuinely curious.